content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -126,6 +126,28 @@ iptables:
 ...
 ```
 
+##### Experimental performance improvements {#minimize-iptables-restore}
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+In Kubernetes 1.27, new performance improvements to the iptables proxy
+mode are enabled by default, which should make it much less necessary to
+override the default syncing timeouts.
```

Comment:
should we add context that this is focused on large environments, as in number of clusters or services?